Antonio_Ramos_Documentation

```diff
@@ -140,12 +148,86 @@ new code - pFC = r"G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\CARR\California\Ba
+1. ZIPS_Tot_Emp.r 
+2. CA_WAC_Transformation.R
+3. ZCTA_Manufa_Trans.R
+4. ZCTA_GoodLabs_Trans.py
+5. ZCTA_Manufa_Trans.py 
+6. GoodLabs_Trans.py
+(more complete descriptions avaliable in Data & Script File Table located at G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin) 
+
+- discovered that the two of the three data files, GoodLabs_New.shp and CA_ZCTA_Man.shp, from the K-function Local_3Stage.py script come from one python script, "ZCTA_GoodLabs_Trans.py" (located at "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\kfunctions\Three Stage K-functions\ZCTA_GoodLabs_Trans.py") 
+
+- ZCTA_GoodLabs_Trans.py changes the CRS for GoodLabs.shp and exports GoodLabs_New.shp 
+
+- ZCTA_GoodLabs_Trans.py also also reads a csv file (DEC_00_SF3_DP3_with_ann.csv) and joins it with an imported shapefile (tl_2010_06_zcta500.shp) changes the CRS and then exports CA_ZCTA_Man.shp
+
+- all other R and python scripts do not seem to be relevant to the data that is currently being used by the K-function Local_3Stage.py script. However, it is not clear where the data file CA_Block_New.shp came from which is the third data file being read into the K-function Local_3Stage.py script. 
+
+- In the folder located at "G:\MAX-Filer\Collab\Labs-kbuzard-S18\Admin\Block Level Analysis" I found two programs that might be related to CA_Block_New.shp 
+ 1. CA_Block_Trans.py
+ 2. CA_WAC_Trans.py 
+both programs are written August 8th but there is no documentation for why 
+
+- CA_Block_Trans.py has a note that says "Need to figure out how to change shapefile from ANSI 1252 to UTF-8 encoding." Not sure if this was ever done will look for more up to date file. After examining the code it looks like CA_Block_Trans.py is the same as CA_WAC_Trans.py except CA_Block_Trans.py changes the geometry and CRS. CA_Block_Trans.py reads in a csv.gz file (ca_wac_S000_JT00_2002.csv.gz) and and joins it with an imported shapefile (CA_block_2000.shp) then changes the geometry and CRS and exports a shapefile (CA_2000_Tot_Emp.shp) however, this is not the block data file which is being read into K-function Local_3Stage.py. 
+
+- found the origin of ca_wac_S000_JT00_2002.csv.gz, https://lehd.ces.census.gov/data/lodes/LODES7/ca/wac/  (ask prof. buzard if she can help clarify what the content of this file is) 
+
+06/13/2018
+- returning to the issue GoodLabs_New.shp not plotting anything when added as a layer to a blank ArcMap map, when I tested GoodLabs.shp the original file, the points do appear plotted on the map. 
+
+-changed projected coordinate system (PCS) for GoodLabs.shp in ArcGis to make it compatibale with the PCS of CA_Block_New (NAD 1983 Contiguous USA Albers) the outputted file with the updated PCS is GoodLabs_Project.shp located with the above mentioned files at "G:\MAX-Filer\Research\Labs-kbuzard-S18\Admin\CARR\California\Base Data" (do you want me to outline the steps for changing the PCS in ArcGis?) 
```

Comment:
Are you not able to change the PCS using Python code? We don't need the steps for doing this manually in ArcGIS if it can be done programmatically, which is the ultimate goal.

Reply:
You should be able to do it in python, and that is what Daniel is investigating.